onnxruntime/core/optimizer/qdq_transformer/selectors_actions/shared/utils.cc

```diff
@@ -35,7 +35,6 @@ static const OpVersionsAndSelector::OpVersionsMap GetMiscOpVersionsMap() {
           {"Transpose", {}},
           {"MaxPool", {12}},
           {"Resize", {}},
-          {"Split", {}},
```

Comment:
This previously used the DropQDQNodeGroupSelector, which only checks that input[0] and output[0] have the same quantization parameters. The other outputs were not checked.

EPs now use the newly added SplitNodeGroupSelector, which by default allows output quantization parameters to differ. This is necessary because the quantization script can generate Q/DQ models in which a Split output has different quantization parameters (e.g. Split -> Clip).

Reply:
Is the main aspect that they're operators that do not change data? Dropping QDQ to execute in fp32 is a potential action.

Reply:
I think the naming for some others could be maybe tweaked as well so the name is more about what is being selected rather than what _might_ be done in an action, which could vary by EP.

I'd settle for a comment that said these were ops that did not change data values, and as such the DQ scale and zero point values are unchanged in the following Q.

Reply:
Added a comment above that says these ops do not change data values.